README.md

```diff
@@ -2,33 +2,55 @@
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/7243b915eb214102be03f7c7b4dd2681)](https://app.codacy.com/gh/physarumAdv/Physarum_webGL?utm_source=github.com&utm_medium=referral&utm_content=physarumAdv/Physarum_webGL&utm_campaign=Badge_Grade_Settings)
 
-Visualization written on JS library THREE.js of a clever mould Physarum Polycephalum
-
-You can view this project [here](http://pavtiger.silaeder.ru/physarum)
-
-## Physarum Polycephalum 
-In nature is a mould that minimizes energy on
-growth process and food transfer. This way it builds the shortes and most optimal ways between points
-
-![Screenshot](readme_images/physarum.gif)
-
-## Cube
-Now growth is visualized at a single cube but mesh can be imported as a .gltf 3D format
-![Screenshot](readme_images/cube.jpg)
-
-## Growth
-![Screenshot](readme_images/mold.jpg)
-
-## Change color of the mold
-Using a slider at the top-right side of the screen you can change the color of the mold to anything you want while simulation is running
-![Screenshot](readme_images/color.jpg)
+This is a visualizer for Physarum Polycephalum's growth written in JavaScript using THREE.js and Node.js libraries.
+We recommend using [this](https://github.com/physarumAdv/minds_crawl) simulator. <br/>
```

Comment:
(minor) I would instead write something slightly different, for example, "The visualizer is developed for this simulator" (or "with" instead of "for"). I'm not sure if it is appropriate for a visualizer to "recommend" a simulator